### PR TITLE
changes for sphinx documentation on windows

### PR DESCRIPTION
**Pull request by rnaskulwar**
changes are specific to windows

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -19,6 +19,11 @@ cmake_minimum_required(VERSION 3.17)
 
 project(Video_Processing_Framework)
 
+if(WIN32)
+    include_directories(${CUDA_TOOLKIT_ROOT_DIR}/include)
+    link_directories(${CUDA_TOOLKIT_ROOT_DIR}/lib/x64)
+endif(WIN32)
```
Comment (theHamsta) [inline CMakeLists.txt:25]: this shouldn't be added manually and without `PRIVATE`/`PUBLIC` specifier. See #397, where I would use https://cmake.org/cmake/help/latest/module/FindCUDAToolkit.html

File: docs/FindSphinx.cmake [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# CMake find_package() Module for Sphinx documentation generator
```
Comment (theHamsta) [inline docs/FindSphinx.cmake:1]: where is this file used? I couldn't find a `find_package(Sphinx REQUIRED)` in this repository.

File: CMakeLists.txt (hunk 1 above)
Comment (rnaskulwar) [reply CMakeLists.txt:25]: changes from local repo got pushed, did not intend to do it.

File: docs/FindSphinx.cmake (hunk 2 above)
Comment (rnaskulwar) [reply docs/FindSphinx.cmake:1]: This file helps to find SPHINX module

Comment (rnaskulwar) [reply docs/FindSphinx.cmake:1]: some how find_package() function does not work on Windows

Comment (theHamsta) [reply docs/FindSphinx.cmake:1]: but where is Sphinx even invoked by CMake? I thought it is purely used by `docs/Makefile`/Python. `FindSphinx` would be useful when we would do something like https://www.vortech.nl/en/integrating-sphinx-in-cmake/ where Sphinx is invoked by CMake. How are you currently generating the documentation? Maybe you missed some changes to `CMakeLists.txt`?

**Comment by theHamsta** [comment]
@rnaskulwar I think this can be closed. Do the instructions in the README for building the docs are sufficient? We could replace them with what the `make html` would do to allow it to work on Windows: https://github.com/NVIDIA/VideoProcessingFramework/blob/6770a888734bf28fa51c72d2d697a1422e6fb77a/docs/Makefile#L56-L58 Integrating this into CMake could be more complicated as it requires to install `PyNvCodec` first.

**Pull request closed without merging**